ai.py

```diff
@@ -247,9 +248,20 @@ async def on_ready():
         asyncio.ensure_future(storage.release_timed(bot))
 
 
+@bot.event
+async def on_command_error(context, error):
+    LOGGER.error(f"!!! Exception caught in {context.command} command !!!")
+    LOGGER.error(error)
+    with open("ai.log", "a") as exception_file:
+        traceback.print_exception(type(error), error, error.__traceback__, file=exception_file)
+    traceback.print_exception(type(error), error, error.__traceback__, file=sys.stderr)
```

Comment:
I would prefer this, because it's all handled with the logger:
```suggestion
    LOGGER.error(f"!!! Exception caught in {context.command} command !!!")
    LOGGER.error(error, exc_info=True, stack_info=True)
```